pkg/pillar/dpcmanager/dpcmanager.go

```diff
@@ -117,7 +117,9 @@ type DpcManager struct {
 	dpcTestBetterInterval time.Duration // Look for lower/better index
 	geoRedoInterval       time.Duration
 	geoRetryInterval      time.Duration
+	enableLastResort      bool
 	lastPublishedLocInfo  types.WwanLocationInfo
+	lastPublishedLocInfo  time.Time
```

Comment:
`lastPublishedLocInfo` is here twice - probably a bad automatic git merge

Reply:
(I have changed the type in one of the previous PRs)

Reply:
Odd I did not do any manual merge here.

Reply:
Fixed